### PR TITLE
Add group create and group delete command in UserGuide

### PR DESCRIPTION
**Pull request by lelerer**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -312,7 +312,49 @@ Example of usage:
...
+> A group represents one or more individuals. It is used as a shortcut in several commands for identifying a group of individual persons.<br>
+
+Format: `group /delete /gid [GROUP_ID]`
+
+* `[GROUP_ID]` refers to the unique identifier of the group.
+    * The unique identifier for a group can be retrieved with [`group /list`](#listing-all-groups-group-list) command.
+
+> **💡 Note(s):**
+>- A group with a unique identifier of `[GROUP_ID]` has to exist before it can be removed.
+
```
Comment (froststein) [inline docs/UserGuide.md:349]: Perhaps add a warning like how `session /delete` does it to warn the user that a delete is irreversible.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -312,7 +312,49 @@ Example of usage:
 <br>
 
 ### Creating a group: `group /create`
+
+> Creates a new group . <br>
```
Comment (froststein) [inline docs/UserGuide.md:316]: Perhaps remove the extra space between group and .

**Review by froststein — APPROVED** [review]
Overall looks good, just needs to fix some spacing and add a warning section for group delete command.

**Review by ivanaitzliddat — CHANGES_REQUESTED** [review]
Looks good overall, just a small change, along with others' comments

**Review by ivanaitzliddat — APPROVED** [review]
Looks good to merge

**Pull request merged**